content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md

```diff
@@ -194,4 +221,3 @@ NodeScore   =  (5 * 5) + (7 * 1) + (10 * 3) / (5 + 1 + 3)
 
 ```
 
```

Comment:
Would it be helpful to add a _What's Next_ link to eg https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/ or https://kubernetes.io/docs/reference/scheduling/ ?

Reply:
makes sense. done.